tests/ert_tests/ert3/config/test_parameters_config.py

```diff
@@ -20,28 +20,51 @@ def test_valid_gauss(mean, std):
                     "std": std,
                 },
             },
-            "variables": ["x", "y", "z"],
         }
     ]
     parameters_config = ert3.config.load_parameters_config(raw_config)
 
     assert len(parameters_config) == 1
     param = parameters_config[0]
 
-    assert param.name == "my_parameter_group"
+    assert param.name == "my_gaussian_parameter"
     assert param.type == "stochastic"
     assert param.distribution.type == "gaussian"
     assert param.distribution.input.mean == mean
     assert param.distribution.input.std == std
-    assert tuple(param.variables) == ("x", "y", "z")
 
     distribution = param.as_distribution()
     assert isinstance(distribution, ert3.stats.Gaussian)
-    assert tuple(param.variables) == tuple(distribution.index)
     assert param.distribution.input.mean == distribution._mean
     assert param.distribution.input.std == distribution._std
 
 
+def test_valid_gauss_variables():
+    raw_config = [
+        {
+            "name": "my_parameter_group",
```

Comment:
Don't use "group" here in the parameter name, as we want to use "parameter grouping" in a different context (a level above).

Reply:
Will leave this to a future PR as the concept "parameter group" is still in the code.